.gitignore

```diff
@@ -2,3 +2,4 @@
 package-lock.json
 persist.txt
 node_modules/*
+dist/*
```

Comment:
I would like to have a built version available in the repo so people can try it out without having to have the build tools installed.

Reply:
I see, hmm... that might be somewhat of an issue with the assets, unless we pack them somehow instead of just copying. (else they would be duplicated)